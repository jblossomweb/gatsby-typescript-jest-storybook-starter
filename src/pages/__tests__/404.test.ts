import snapshotTests from "../../../utils/story-snapshot-tests"

jest.mock("../../templates/default")
jest.mock("../../components/seo")

import { stories } from "./404.stories"

describe("pages/404", () => {
  snapshotTests(stories)
})
