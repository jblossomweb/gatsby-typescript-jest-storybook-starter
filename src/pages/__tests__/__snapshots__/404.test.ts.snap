// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/404 sample matches snapshot 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          Test Title
        </a>
      </h1>
    </div>
  </header>,
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0 1.0875rem 1.45rem",
      }
    }
  >
    <main>
      <h1>
        NOT FOUND
      </h1>
      <p>
        You just hit a route that doesn't exist... the sadness.
      </p>
    </main>
    <footer>
      Â© 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        target="_blank"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;
